lib/raven/instance.rb

```diff
@@ -58,7 +67,6 @@ def report_status
         logger.info "Raven #{VERSION} configured not to send errors."
       end
     end
-    alias_method :report_ready, :report_status
```

Comment:
`Forwardable` doesn't support forwarding to aliases, (but you can set up an alias at the top level that forwards to the same underlying method) so all the aliases were moved up to `Base`. That also kept all the aliases consolidated, which seemed a bit cleaner if more needed to be added in the future.